mne_bids_pipeline/_config.py

```diff
@@ -933,12 +933,7 @@
     ```
 """
 
-contrasts: Iterable[
-    Union[
-        Tuple[str, str],
-        ArbitraryContrast
-    ]
-] = []
+contrasts: Iterable[Union[Tuple[str, str], ArbitraryContrast]] = []
```

Comment:
So difficult to read. Time to drop support for Python <3.10 so we can use `X | Y` instead of `Union[X, Y]` 😎

No gut really, here we'd really gain a usability boost from a switch... maybe next spring or so??

Reply:
My vote is to go with whenever MNE-Python switches to 3.10 as the min based on EOL